### PR TITLE
[MAINTENANCE] Remove runme fixtures/stages and enable docs-integration to run automatically

### PR DESCRIPTION
**Pull request by kenwade4**
Changes proposed in this pull request:
- Remove runme from fixtures
- Remove runme ci stage
- Enable docs-integration pipeline to run automatically

### CODE REVIEW

File: ci/azure-pipelines-docs-integration.yml [hunk 1]
```diff
@@ -58,14 +56,15 @@ stages:
                 docs/**
                 tests/integration/docusaurus/**
                 tests/integration/fixtures/**
+                tests/integration/test_definitions/**
+                tests/integration/test_script_runner.py
```
Comment (anthonyburdi) [inline ci/azure-pipelines-docs-integration.yml:60]: Nice catch!

**Review by anthonyburdi — APPROVED** [review]
LGTM! In the future we might want to add the manual trigger as an `or` i.e. runs on all PRs and also when we want to manually run it. But not necessary for this PR. Thanks for turning these tests back on!

File: ci/azure-pipelines-docs-integration.yml (hunk 1 above)
Comment (kenwade4) [reply ci/azure-pipelines-docs-integration.yml:60]: Yea for sure... it's nice to manually trigger when needed

Comment (kenwade4) [reply ci/azure-pipelines-docs-integration.yml:60]: But we are gonna talk about a bunch of potential pipeline changes at the arch meeting tomorrow. I think we should make all/most pipelines and stages manually runnable.

**Pull request merged**